stock_request_analytic/views/analytic_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2016-2020 ForgeFlow, S.L. (https://www.forgeflow.com)
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="view_account_analytic_account_form" model="ir.ui.view">
+        <field name="name">analytic.order.form</field>
+        <field name="model">account.analytic.account</field>
+        <field name="inherit_id" ref="analytic.view_account_analytic_account_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='main']" position='after'>
+                <notebook groups="stock_request.group_stock_request_user">
+                    <page string="Stock Request" name="stock_request">
+                        <field name="stock_request_ids" />
+                    </page>
+                </notebook>
+            </xpath>
+        </field>
```

Comment:
One suggestion, could you move this to a smart button? this list can get huge quite quckly

Reply:
Thank you for your suggestion. I already update this code. Could you review it again?